internal/strict/strict.go

```diff
@@ -103,6 +106,10 @@ var StrictControls = Controls{
 		Error:   errors.New("The `validate-all` command is no longer supported. Use `terragrunt run-all validate` instead."),
 		Warning: "The `validate-all` command is deprecated and will be removed in a future version. Use `terragrunt run-all validate` instead.",
 	},
+	SkipDependenciesInputs: {
+		Error:   errors.New("The `skip-dependencies-inputs` is no longer supported. Dependencies inputs handling will be changed."),
```

Comment:
Recommended: Reading inputs from dependencies has been deprecated. It will be removed in a future version of Terragrunt. To continue to use inputs from dependencies forward them as outputs.